other/materials_designer/create_interface_with_min_strain_zsl.ipynb

```diff
@@ -46,13 +46,11 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    Interface.from_str(str, "POSCAR")
</pre><p>What's all the above in this cell??</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>